pygac/calibration.py

```diff
@@ -41,9 +41,11 @@ class Calibrator(object):
     Calibrator = namedtuple('Calibrator', fields)
     Calibrator.__new__.__defaults__ = (None,) * len(fields)
 
-    def __new__(cls, spacecraft):
+    def __new__(cls, spacecraft, custom_coeffs=None):
+        custom = custom_coeffs or {}
+        default = default_coeffs[spacecraft]
         spacecraft_coeffs = {
-            key: cls.parse(coeffs[spacecraft].get(key))
+            key: cls.parse(custom.get(key) or default.get(key))
             for key in cls.fields
```

Comment:
```suggestion
            **default, **custom
```

Reply:
In principle this take default values from `default` and uses `custom` coeffs when availble

Reply:
This is only available for python 3.5+, so it won't survive the CI checks for python 2.7... The backward compatible way I found is:
```python
spacecraft_coeffs = default.copy()
spacecraft_coeffs.update(custom)
```
In my opinion, we can do without python 2 compatibility, but I do not know if it is also your intention.